paramfetch_test.go

```diff
@@ -82,3 +84,29 @@ func TestGetParamsParallel(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+func TestCheckFileIgnoresNonVKExtension(t *testing.T) {
+	const mockParamInfoBytes = `{
+			"cid": "Qmxxxxxdoesntexist",
+			"digest": "0e0958009936b9d5e515ec97b8cb792d",
+			"sector_size": 2048
+	}`
+	var mockParamInfo paramFile
+	if err := json.Unmarshal([]byte(mockParamInfoBytes), &mockParamInfo); err != nil {
+		require.NoError(t, err)
+	}
+	ft := &fetch{}
+
+	err := os.Setenv("TRUST_PARAMS", "1")
+	require.NoError(t, err)
+	defer func() {
+		err := os.Unsetenv("TRUST_PARAMS")
+		assert.NoError(t, err)
+	}()
```

Comment:
I'm not convinced this won't leak into other test contexts. Is there a way to manage envvars in a more isolated way? Or do we think this is sufficient?

Reply:
should be fine